activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+*   `after_commit` callbacks defined on models now execute in the correct order.
+
+    ```ruby
+    class User < ActiveRecord::Base
+      after_commit { puts("this gets called first") }
+      after_commit { puts("this gets called second") }
+    end
+    ```
+
+    Previously, the callbacks executed in the reverse order. To opt in to the new behaviour:
+
+    ```ruby
+    config.active_record.run_after_transaction_callbacks_in_order_defined = true
+    ```
+
+    This is the default for new apps.
```

Comment:
I was confused by this language. To me it implied that the default behaviour was unchanged and in order to opt-in to this new behavuour I had to set this config to true. So just reading the changelog I decided not to update this config.

Then in one of my models with Active Storage integration, the files stopped getting uploaded. Looking deeper I found that an after_commit callback of mine was reloading the object before Active Storage could upload it in its after_commit hook. There was no exception.